R/get_uid.R

```diff
@@ -75,13 +75,34 @@
 #' }
 
 get_uid <- function(sciname, ask = TRUE, verbose = TRUE, rows = NA){
+  repeat_until_it_works <- function(func, catch, max_tries = 3, wait_time = 10, verbose = TRUE, ...) {
```

Comment:
Should we pass these arguments (max_tries, wait_time, etc) from the main function? Maybe by adding dots to `get_uid`?